src/app/app-routing.module.ts

```diff
@@ -11,7 +11,7 @@ import { UserComponent } from './shared/user/user.component';
 
 const routes: Routes = [
   { path: '', redirectTo: '/explore', pathMatch: 'full' },
-  { path: 'settings', loadChildren: () => import('./settings/settings.module').then(m => m.SettingsModule) },
+  { path: 'settings/:id', loadChildren: () => import('./settings/settings.module').then(m => m.SettingsModule) },
```

Comment:
why we need a settingsId?

Reply:
settings/:userID

Reply:
I think what we want is base/:userId/settings